backend/src/repositories/review.repository.ts

```diff
@@ -1,42 +1,60 @@
-import { ReviewEntity } from "../entity/Review";
-import { EntityManager, In } from "typeorm";
+import { PrismaClient, reviews } from "@prisma/client";
+import { PostReviewRequestBody } from "../api/schemas/review.schema";
 
 export class ReviewRepository {
-  constructor(private readonly manager: EntityManager) {}
+  constructor(private readonly prisma: PrismaClient) {}
 
-  async getAllReviews(): Promise<ReviewEntity[]> {
-    return await this.manager.find(ReviewEntity);
+  async getAllReviews(): Promise<reviews[]> {
+    return await this.prisma.reviews.findMany();
   }
 
-  async getCourseReviews(courseCode: string): Promise<ReviewEntity[]> {
-    return await this.manager.find(ReviewEntity, {
+  async getCourseReviews(courseCode: string): Promise<reviews[]> {
+    return await this.prisma.reviews.findMany({
       where: {
-        courseCode,
+        courseCode: courseCode,
```

Comment:
O_o

Reply:
bro pleaseeeeee 😭😭😭 desperate debugging calls for desperate measures, pretend u didnt see this